lib/fastlane/plugin/appcenter/helper/appcenter_helper.rb

```diff
@@ -136,10 +137,11 @@ def self.create_dsym_upload(api_token, owner_name, app_name)
       end
 
       # commits or aborts symbol upload
-      def self.update_symbol_upload(api_token, owner_name, app_name, symbol_upload_id, status)
+      def self.update_symbol_upload(api_token, owner_name, app_name, symbol_upload_id, status, timeout)
```

Comment:
On the other hand, this is not uploading anything, merely doing a quick PATCH call to the API to signal that the upload has completed. So this timeout is not necessary.